docs/operational_guide/availability_consistency_durability.md

```diff
@@ -0,0 +1,126 @@
+
+## Tuning for Performance and Availability
+
+### Client Write and Read consistency
+
+We recommend running the client with `writeConsistencyLevel` set to `majority` and `readConsistencyLevel` set to `unstrict_majority`.
+This means that all write must be acknowledged by a quorums of nodes in order to be considered succesful, and that reads will attempt to achieve quorum, but will return the data from a single node if they are unable to achieve quorum.
+You can read about the consistency levels in more detail in [the Consistency Levels section](../m3db/architecture/consistencylevels.md)
+
+### Commitlog Configuration
+
+We recommend running M3DB with an asynchronous commitlog.
+This means that writes will be reported as successful by the client, though the data may not have been flushed to disk yet.
+
+For example, consider the default configuration:
+
+```
+commitlog:
+  flushMaxBytes: 524288
+  flushEvery: 1s
+  queue:
+    calculationType: fixed
+    size: 2097152
+```
+
+This configuration states that the commitlog should be flushed whenever either of the following is true:
+
+1. 524288 or more bytes have been written since the last time M3DB flushed the commitlog.
+2. One or more seconds has elapsed since the last time M3DB flushed the commitlog.
+
+In addition, the configuration also states that M3DB should allow up to `2097152` writes to be buffered in the commitlog queue before the database node will begin rejecting incoming writes so it can attempt to drain the queue and catch up. Increasing the size of this queue can often increase the write throughput of an M3DB node at the cost of potentially losing more data if the node experiences a sudden failure like a hard crash or power loss.
+
+### Writing New Series Asynchronously
+
+The default M3DB YAML configuration will contain the following as a top-level key under the `db` section:
+
+```
+writeNewSeriesAsync: true
+```
+
+This instructs M3DB to handle writes for new timeseries (for a given time block) asynchronously. Creating a new timeseries in memory is much more expensive than simply appending a new write to an existing series, so the default configuration of creating them asynchronously improves M3DBs write throughput significantly when many new series are being created all at once.
+
+However, since new time series are created asynchronously, its possible that there may be a brief delay inbetween when a write is acknowledged by the client and when that series becomes available for subsequent reads.
+
+M3DB also allows operators to ratelimit the number of new series that can be created per second via the following configuration:
+
+```
+writeNewSeriesLimitPerSecond: 1048576
+```
+
+This value can be set much lower than the default value for workloads in which a significant increase in cardinality usually indicates an abusive caller.
```

Comment:
> abusive caller

i'd recommend changing this to "misbehaving", as abuse implies malicious intent. more likely, it's a caller that doesn't understand the goals or limitations of the system.

don't want to get too tied up in semantics, but if we use words like "abusive", "misbehaving", etc, we might want to very clearly call out or link to expectations (e.g. wrt dimensionality, cardinality, etc).

Reply:
Misbehaving in this context would depend on your setup and workload. I.E at Uber it might be someone emitting UUIDs in a spark job, but for someone elses setup that might be the intended use case

Reply:
understood - still think we should switch to "misbehaving" instead of "abusive" unless you disagree with the semantics there.

Reply:
yep i did